### PR TITLE
New py2 dlc dockerfiles

### PR DESCRIPTION
**Pull request by saimidu**
Create Dockerfile.eia for 1.4.1 with py2, with health check binary included

Tests

- Tested by serving squeezenet model on container.
- Tested with Sagemaker integration tests - "batch_transform.py", " 	test_elastic_inference.py", "test_hosting.py".

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/1.4.1/py2/mms-entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import shlex
```
Comment (surajkota) [inline docker/1.4.1/py2/mms-entrypoint.py]: remove the old entrypoint script

File: docker/1.4.1/py2/Dockerfile.eia [hunk 2]
```diff
@@ -1,21 +1,50 @@
...
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python
+ARG PIP=pip
+ARG HEALTH_CHECK_VERSION=1.3.2
```
Comment (surajkota) [inline docker/1.4.1/py2/Dockerfile.eia]: does this need to be updated to 1.3.3. I see in TF dockerfiles
https://github.com/aws/sagemaker-tensorflow-serving-container/pull/77/files#diff-579c54887ad8272b572b526bc4d10f5eR10

File: docker/1.4.1/py2/Dockerfile.eia [hunk 3]
```diff
@@ -1,21 +1,50 @@
-FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py2.7
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
```
Comment (surajkota) [inline docker/1.4.1/py2/Dockerfile.eia]: remove extra space

File: docker/1.4.1/py2/mms-entrypoint.py (hunk 1 above)
Comment (saimidu) [reply docker/1.4.1/py2/mms-entrypoint.py]: The old entrypoint also uses config.properties, but the python one doesn't. Should I also remove that?

File: docker/1.4.1/py2/Dockerfile.eia (hunk 2 above)
Comment (saimidu) [reply docker/1.4.1/py2/Dockerfile.eia]: Updated to 1.3.3.

Thanks.

File: docker/1.4.1/py2/mms-entrypoint.py (hunk 1 above)
Comment (surajkota) [reply docker/1.4.1/py2/mms-entrypoint.py]: Sai and I looked at existing cpu and gpu docker files and found that they use the shell script. So we keep it for now until we update the cpu and gpu images for py2

Comment (mvsusp) [inline docker/1.4.1/py2/mms-entrypoint.py]: Add copyright on the top of the file

Comment (saimidu) [reply docker/1.4.1/py2/mms-entrypoint.py]: I don't see a sample for a copyright in the master branch.

**Pull request merged**